### PR TITLE
Add binder links

### PR DESCRIPTION
**Pull request by jasmainak**
@larsoner nerdsniped me ... I hope this works in 2 hours :)

cc @choldgraf it may interest you

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -414,14 +414,19 @@ def reset_warnings(gallery_conf, fname):
     'junit': op.join('..', 'test-results', 'sphinx-gallery', 'junit.xml')
 }
 
+build_dev_html = False
 if not build_dev_html:
     sphinx_gallery_conf.update({
-        'binder': {'org': 'mne-tools',
+        'binder': {'org': 'jasmainak',
```
Comment (agramfort) [inline doc/conf.py]: on purpose?

**Comment by larsoner** [comment]
I went to CircleCI

https://12832-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/plot_source_alignment.html#alignment-without-mri

Clicked "Launch Binder", and while it was loading, saw a big "404 : Not Found" error at the bottom in the "Here's a non-interactive preview on nbviewer while we start a server for you" area:

Screen Shot 2019-04-27 at 00 01 19

Then after 20 minutes or so it said "launching server" in the log and gave up with "404 : Not Found
You are requesting a page that does not exist!":

Screen Shot 2019-04-27 at 00 16 01

Does it work for you?

**Comment by jasmainak** [comment]
Ah yes, it won't work. We would also have to add the notebooks/ directory. Otherwise it wasn't working.

It's a bit weird that sphinx gallery is not smart enough to re-use the notebooks that are available for download (in the _downloads directory). But instead the notebooks are copied to a separate directory. This means there are two copies of the notebook and very likely also two code paths in sphinx-gallery.

Do you think it might be reasonable to add the notebooks/ directory for the stable build? Or do you think we should fix sphinx gallery again?

**Comment by larsoner** [comment]
Let's open a Binder issue. It would be nice if we did not have to have two copies of notebooks, and two copies of the `Dockerfile`, and if these did not have to live at the HTML root directory.

**Comment by jasmainak** [comment]
@larsoner based on the discussion here: https://github.com/sphinx-gallery/sphinx-gallery/issues/483 how should we proceed? It seems to me that there are multiple things that need to be fixed upstream before it can be adapted. One is the duplication of notebooks and the other is the location in which the Dockerfile lives. 

Even if we manage to fix that, I realized that because of the one repo, one mybinder rule this would work only for one version. Thus, when you have a new version you have to rebuild the docs not just for the current version but for the older version to remove the links.

**Comment by jasmainak** [comment]
alright!

**Pull request closed without merging**